assets/js/src/core/components/tag/tag.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { Tag as AntTag, type TagProps as AntTagPropsProps } from 'antd'
+import React from 'react'
+import { useStyles } from './tag.styles'
+import { Icon } from '@Pimcore/components/icon/icon'
+import { type LiteralUnion } from 'antd/es/_util/type'
+import type { PresetColorType, PresetStatusColorType } from 'antd/es/_util/colors'
+
+export interface TagProps extends AntTagPropsProps {
+  text: string
+  iconName?: IconNameType
```

Comment:
hmm, why do we have this limitation of icons?
I would rather accept any string. Otherwise the tag component is not really flexible.

Reply:
@vin0401 
My thought behind it was that it would be good to have some kind of feedback when an icon name does not yet exist. 
I played a bit with using the IconLibrary to determine which strings are allowed but Storybook does not like me using the injection in this way :P 

<img width="850" alt="Screenshot 2024-09-23 at 13 33 49" src="https://github.com/user-attachments/assets/0ca3f5d2-ba3d-4348-b43b-3fd75e94785b">

Have made it a n open string in coming push